### PR TITLE
Note in “Private class features”: private class field really private

### LINKED ISSUE #12992: Content bug: inaccurate information about Private Class Fields
## What page(s) did you find the problem on?
[Private_class_fields](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields)

## Specific page section or heading?

[Examples > Private Fields](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields#private_fields) > code example:

```js
class ClassWithPrivateField {
  #privateField;

  constructor() {
    this.#privateField = 42;
  }
}

class SubClass extends ClassWithPrivateField {
  #subPrivateField;

  constructor() {
    super();
    this.#subPrivateField = 23;
  }
}

new SubClass();
// SubClass {#privateField: 42, #subPrivateField: 23}
```

## What is the problem?

Base Class Private Fields are not accessible from Sub Classes as the code example implies.
```// SubClass {#privateField: 42, #subPrivateField: 23}```

## What did you expect to see?

Explain that private fields are not accessible from the sub class, however public fields are.
`// SubClass {#subPrivateField: 23}`

## Did you test this? If so, how?

```js
class ClassWithPrivateField {
  publicField
  #privateField
  constructor() {
    this.#privateField = 42
    this.publicField = 69
  }
}

class SubClass extends ClassWithPrivateField {
  subPublicField
  #subPrivateField
  constructor() {
    super()
    this.#subPrivateField = 23
    this.subPublicField = 10
  }
  get publicFields(){return [this.publicField, this.subPublicField]}
  get privateFields(){return [this.#privateField, this.#subPrivateField]}
}
const test = new SubClass()
console.log(test.publicFields) // Array [69, 10]
console.log(test.privateFields) // Error: reference to undeclared private field or method #privateField
```

Edit: I added a couple of working examples:
[JSFiddle - Sub Class with Public Class Fields](https://jsfiddle.net/SubZ390/xvn0k542/4/)
[JSFiddle - Sub Class with Private Class Fields](https://jsfiddle.net/SubZ390/un3fx097/1/)

### PR DESCRIPTION
**Pull request by sideshowbarker**
Fixes https://github.com/mdn/content/issues/12992

### CODE REVIEW

File: files/en-us/web/javascript/reference/classes/private_class_fields/index.md [hunk 1]
```diff
@@ -96,9 +96,12 @@ class SubClass extends ClassWithPrivateField {
 }
 
 new SubClass();
-// SubClass {#privateField: 42, #subPrivateField: 23}
+// SubClass {#subPrivateField: 23}
+private accessible
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/classes/private_class_fields/index.md:100]: What is this line for?

Comment (sideshowbarker) [inline files/en-us/web/javascript/reference/classes/private_class_fields/index.md:100]:
```suggestion
```

Comment (sideshowbarker) [reply files/en-us/web/javascript/reference/classes/private_class_fields/index.md]: No idea but I guess it got added somehow when I was editing the doc. Maybe through some auto-completion thing in my editor, I dunno. Anyway, gone now

**Review by teoli2003 — APPROVED** [review]
Thanks!

**Pull request merged**